flamedisx/er_nr_base.py

```diff
@@ -723,6 +740,7 @@ def _simulate_nq(self, energies):
         return np.floor(energies / work).astype(np.int)
 
 
+
```

Comment:
Spurious newline

Reply:
Removed spurious new line. So 2 blank lines between 2 consecutive classes in the same .py?